assets/less/zenodo-rdm/elements/segment.overrides

```diff
@@ -28,4 +28,16 @@
       padding: 0;
     }
   }
+}
+
+.ui.segments.very.rounded {
```

Comment:
are you trying to decrease the "roundiness" of the element? in this case wouldn't it be enough to remove "very" class?

Reply:
no, the opposite 😄 I'm increasing it, and tried to match the equivalent class-naming for padded segments (`.ui.padded.segment` & `.ui.very.padded.segment`)